integration/common_test.go

```diff
@@ -81,6 +81,9 @@ func (s *RunSuite) TearDownTest(c *C) {
 	containers, err := client.ListContainers(dockerclient.ListContainersOptions{All: true})
 	c.Assert(err, IsNil)
 	for _, container := range containers {
+		// Unpause container (if paused) and ignore error (if wasn't paused)
+		client.UnpauseContainer(container.ID)
```

Comment:
As commented, this is to make sure we won't fail in `TearDown` because of a paused container… So we're trying to unpause all of them and move on :wink:.